staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go

```diff
@@ -568,10 +584,12 @@ func TestMuxAndDiscoveryComplete(t *testing.T) {
 	}
 
 	// start the API server
-	stopCh, runCompletedCh := make(chan struct{}), make(chan struct{})
+	runCompletedCh := make(chan struct{})
```

Comment:
same

Reply:
Changed.

It's perhaps worth adding that instead of needing three lines of code, ktesting would only be one (`tCtx := ktesting.Init(t)`) because it includes setting up cancellation.

There's no need for a TODO, finding these `ktesting.NewTestContext(t)` calls will be easy.

Reply:
https://github.com/kubernetes/kubernetes/pull/122148/files/0a89572aed51477ef9baf4963fc29aa2f1a42d51#r1576754590